docs/tutorials/prowler-app-s3-integration.md

```diff
@@ -0,0 +1,334 @@
+                "arn:aws:s3:::<BUCKET NAME>"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+???+ note
+    Replace `<BUCKET AWS ACCOUNT NUMBER>` with the AWS account ID that owns the destination S3 bucket, and `<BUCKET NAME>` with the actual bucket name.
+
+## Step 1: Navigate to Integrations
+
+To access S3 Integration:
+
+1. Navigate to  "Integrations"
+
+    ![Navigate to integrations](../tutorials/img/s3-integration-ui-1.png)
+
+2. The integrations page displays the available integrations
+
+## Step 2: Access S3 Integration Management
+
+1. Locate the Amazon S3 Integration card on the integrations page
+
+2. Click on the "Configure" button on the S3 integration card to access the dedicated management interface
+
+    ![Access S3 integration](../tutorials/img/s3-integration-ui-2.png)
+
+3. Review the S3 integration features and capabilities displayed on the page
+
+## Step 3: Add New S3 Integration
+
+### Configuration Step
+
+1. Click the "Add Integration" button to start the configuration process
+
+    ![Add integration button](../tutorials/img/s3-integration-ui-3.png)
+
+2. Complete the configuration form with the following details:
+
+    - **Cloud Providers:** Select the providers whose scan results should be exported to this S3 bucket
+    - **Bucket Name:** Enter the name of your target S3 bucket (e.g., `my-security-findings-bucket`)
+    - **Output Directory:** Specify the directory path within the bucket (e.g., `/prowler-findings/`)
+
+    ![Configuration form](../tutorials/img/s3-integration-ui-4.png)
+
+3. Click "Next" to proceed to the credentials step
+
+### Credentials Step
+
+1. Configure AWS authentication using one of the supported methods:
+
+    - **AWS SDK Default:** Use default AWS credentials from the environment
+    - **Access Keys:** Provide AWS access key ID and secret access key
+    - **IAM Role (optional):** Specify role ARN, external ID, and optional session parameters
+
+    ![Credentials configuration](../tutorials/img/s3-integration-ui-5.png)
+
+2. For IAM role authentication, complete the required fields:
+
+    - **Role ARN:** The Amazon Resource Name of the IAM role
+    - **External ID:** Unique identifier for additional security (defaults to Tenant/Organization ID)
+    - **Role Session Name:** Optional name for the assumed role session
+    - **Session Duration:** Optional duration in seconds for the session
+
+**Prowler App** provides Infrastructure as Code (IaC) templates to automate the required permissions setup. These templates simplify the process of adding S3 integration permissions to existing Prowler SaaS roles.
+
+### Available Templates
+
+Choose from the following deployment options:
+
+- [CloudFormation](https://raw.githubusercontent.com/prowler-cloud/prowlerpro-permissions-templates-public/main/terraform/s3-integration/main.tf)
+- [Terraform](https://s3.eu-west-1.amazonaws.com/prowler-pro-saas-pro-artifacts/templates/prowler-pro-scan-role-s3-integration.yaml)
```

Comment:
These two links will be ready once we merge https://github.com/prowler-cloud/prowler/pull/8395. These ones are for Prowler SaaS.

Reply:
Done.